composer.json

```diff
@@ -59,11 +60,13 @@
         ],
         "check:composer:config": "Build/Scripts/runTests.sh -n -s composerNormalize",
         "check:php": [
+            "@check:php:lint",
```

Comment:
```suggestion
            "@check:php:rector",
```

Reply:
At least locally this does not work

I have to run check without ddev and rector with ddev

Reply:
Okay, will check later.